### PR TITLE
RC1 - 0.32.4

### PR DESCRIPTION
**Pull request by marbar3778**
- Release candidate for v0.32.4

once this is finalized, will branch off and create the release branch and we can close this.
- waiting on release of tm-db

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+
+Special thanks to external contributors on this release: @jon-certik
+
+Friendly reminder, we have a [bug bounty
+program](https://hackerone.com/tendermint).
+
+### IMPROVEMENTS:
+
+- [rpc] [\#2010](https://github.com/tendermint/tendermint/issues/2010) Add NewHTTPWithClient and NewJSONRPCClientWithHTTPClient (note these and NewHTTP, NewJSONRPCClient functions panic if remote is invalid) (@gracenoah)
+- [rpc] [\#3984](https://github.com/tendermint/tendermint/issues/3984) Add `MempoolClient` interface to `Client` interface
```
Comment (melekes) [inline CHANGELOG.md]: move to breaking section

**Comment by melekes** [comment]
update version/version.go?

File: version/version.go [hunk 2]
```diff
@@ -20,10 +20,10 @@ const (
 	// Must be a string because scripts like dist.sh read this file.
 	// XXX: Don't change the name of this variable or you will break
 	// automation :)
-	TMCoreSemVer = "0.32.3"
+	TMCoreSemVer = "0.32.4"
 
 	// ABCISemVer is the semantic version of the ABCI library
-	ABCISemVer  = "0.16.1"
+	ABCISemVer  = "0.16.2"
```
Comment (marbar3778) [inline version/version.go]: bumped `abci` due to omitempty PR

Comment (melekes) [reply version/version.go]: 😕 ABCI itself has not changed

**Pull request merged**